pom.xml

```diff
@@ -504,6 +504,42 @@
 					<goals>deploy</goals>
 				</configuration>
 			</plugin>
+
+			<!-- Aggregate JavaDoc for use with autoCompleteResponder -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-javadoc-plugin</artifactId>
+				<version>3.1.0</version>
+				<extensions>true</extensions>
+				<executions>
+					<execution>
+						<phase>generate-resources</phase>
+						<goals>
+							<goal>javadoc</goal>
+						</goals>
+					</execution>
+				</executions>
+
+				<configuration>
+					<doclet>nl.praegus.doclets.FixtureDocGenerator</doclet>
+					<docletArtifact>
+						<groupId>nl.praegus</groupId>
+						<artifactId>fixture-doc-generator</artifactId>
+						<version>1.0</version>
+					</docletArtifact>
+					<excludePackageNames>nl.hsac.fitnesse.sample:
```

Comment:
Does the plugin also support an 'include' configuration? Just specifying what to include seems more compact, and less error-prone, that listing everything to exclude.

Reply:
There is a SourceFileIncludes option: https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#sourceFileIncludes

Would have to look into that, but I think we can specify patterns of classes (but not packages) to include.

Reply:
Being able to specify patterns of classes to include (similar to the current `excludePackageNames`) would be my preference.